### PR TITLE
Version 1.9.3 ❄️

### PR DESCRIPTION
**Pull request by SC426**
[Upstream
](https://github.com/ramonhagenaars/typish/tree/v1.9.3)[Changelog](https://github.com/ramonhagenaars/typish/releases)

channel: snowflake

## Actions
- Add as new submodule
- Remove `noarch`
- Linter fixes

### CODE REVIEW

**Review by ryanskeith — APPROVED** [review]
LGTM

**Comment by lorepirri** [comment]
@SC426 could you please reactivate the CI? the build/checks were not triggered (remove `[skip ci]` from the title)

**Comment by lorepirri** [comment]
[PKG-3567]

[PKG-3567]: https://anaconda.atlassian.net/browse/PKG-3567?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Review by lorepirri — APPROVED** [review]
LGTM, thanks!

**Comment by lorepirri** [comment]
@SC426  

- you could add a branch protection rule requiring two reviewers here (for `main`): https://github.com/AnacondaRecipes/typish-feedstock/settings/branches

- and check the `Automatically delete head branches ` here: https://github.com/AnacondaRecipes/typish-feedstock/settings

- please next time the ticket number either in the description (possibly), either in the branch name

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,16 +10,16 @@ source:
...
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
+    - wheel
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:20]:
```suggestion
    - wheel
    - setuptools
```

**Review by JeanChristopheMorinPerso — CHANGES_REQUESTED** [review]

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Pull request closed without merging**